bundle/Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryserviceplugin.js

```diff
@@ -297,6 +298,83 @@ YUI.add('cof-createcontent-universaldiscoveryserviceplugin', function (Y) {
         _deleteContent: function (event) {
             event.content.delete({api: this.get('host').get('capi')}, function () {});
         },
+
+        /**
+         * changeLanguage event handler. It opens languageSelectionBox for selecting
```

Comment:
The description can be simplified.
`Opens the LanguageSelectionBox view instance that allows to select a created content language.`